dotnet-desktop-guide/net/wpf/controls/styles-templates-overview.md

```diff
@@ -232,7 +232,7 @@ To share a set of resources, including styles and templates, across apps, you ca
 
 It is the sharing of `shared.xaml`, which itself defines a <xref:System.Windows.ResourceDictionary> that contains a set of style and brush resources, that enables the controls in an app to have a consistent look.
 
-For more information, see [Merged resource dictionaries](../../../framework/wpf/advanced/merged-resource-dictionaries.md).
+For more information, see [Merged resource dictionaries](/dotnet/desktop/wpf/systems/xaml-resources-merged-dictionaries).
```

Comment:
This doesn't seem right since the article exists in the .net 6 area: `../systems/xaml-resources-merged-dictionaries.md`.

Reply:
@adegeo ready for a round 2 review